DotnetCLIVersion.txt

```diff
@@ -1 +1 @@
-2.0.0-preview1-005724
+2.0.0-preview1-005977
```

Comment:
Source-build is using 2.0.0-preview3-006845

Reply:
For both this and the above, I got the versions from https://github.com/dotnet/source-build/blob/dev/release/2.0/patches/standard/0002-Unify-CLI-and-BuildTools-Versions.patch